bodhi/server/consumers/automatic_updates.py

```diff
@@ -141,10 +141,9 @@ def __call__(self, message: fedora_messaging.api.Message) -> None:
                 stable_karma=3,
                 unstable_karma=-3,
                 user=user,
+                status=UpdateStatus.testing,
             )
 
-            log.debug("Setting request for new update.")
-            update.set_request(dbsession, UpdateRequest.testing, owner_name)
```

Comment:
`set_request` is also sending notifications about update. Shouldn't we sent one here now?

Reply:
The notification was there to notify the update owner that the update was ready to be pushed to testing. This is not valid any more since we are going to create the update in the testing status directly.

If we feel the need to notify the user that the update was automatically created we can maybe add a comment to the update.

Reply:
@cverna an opinion on this one?

Reply:
So thinking about it last night :-), I think we need to add comment to the update so that we notify the packager that an update was automatically created. Otherwise I think that would be silent.

See https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/scripts/approve_testing.py#L98 for an example on how to add an update comment

Reply:
+1 on having a notification, it seems that comment from "bodhi", "autoqa" and "taskotron" aren't sending notifications though, (bodhi/server/models.py line 3096), not quite sure how to go around this :(

Reply:
I went with "anonymous" for now to see how it looks, happy to change it

Reply:
> +1 on having a notification, it seems that comment from "bodhi", "autoqa" and "taskotron" aren't sending notifications though, (bodhi/server/models.py line 3096), not quite sure how to go around this :(

I am pretty sure that `bodhi` user commenting on an update will send a email to the owner of the update. I am not sure we need the fedora-messaging here. 
Do we ?

Reply:
Hm, indeed it does email, just not fedora-messaging.
I kinda think I would like to have fedora-messaging there but this can wait for later.

So it means: changing back anonymous to bodhi and removing all the tests testing fedora-messaging, right?